### PR TITLE
Disable OTP keyboard suggestion on iOS

### PR DESCRIPTION
**Pull request by i1skn**
### Description

On verification screen we do not want to see OTP keyboard suggestion on iOS just yet

### Tested

iOS device

### Backwards compatibility

Yes

### CODE REVIEW

**Review by jeanregisser — COMMENTED** [review]
Erf, and what happens if `keyboardType` is `number-pad`, and `textContentType` set to `username`?
The linked article seems to suggest it disables OTP in that case.

**Comment by i1skn** [comment]
@jeanregisser  It does work indeed! I think I messed up something yesterday when was trying and thought it does not. Thanks for raising this! 🎉

**Review by jeanregisser — APPROVED** [review]
Great! 🚢 👍

File: packages/mobile/src/components/CodeInput.tsx [hunk 1]
```diff
@@ -114,6 +119,7 @@ export default function CodeInput({
 
             {showInput ? (
               <TextInput
+                textContentType="username"
```
Comment (jeanregisser) [inline packages/mobile/src/components/CodeInput.tsx:122]: oh actually let's only do this when `shortVerificationCodesEnabled` is true, right?

**Pull request merged**